common/src/main/resources/assets/wynntils/urls.json

```diff
@@ -92,6 +92,11 @@
     "md5": "f8d2b16d8cd33181b6a3857fa5e390a9",
     "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/combatlocations.json"
   },
+  {
+    "id": "dataStaticCaveInfo",
+    "md5": "ac509a4e620ac65718a6b0816e848db5",
+    "url": "https://raw.githubusercontent.com/Wynntils/Reference/cave_data/content/cave.json"
```

Comment:
Don't forget to use main here.

PR: https://github.com/Wynntils/Reference/pull/26

Reply:
It is merged so you can update now.